apisix/http/router/radixtree_host_uri.lua

```diff
@@ -78,10 +78,11 @@ local function push_host_router(route, host_routes, only_uri_routes)
         vars = route.value.vars,
         filter_fun = filter_fun,
         handler = function (api_ctx, match_opts)
+            local matched = core.table.deepcopy(match_opts.matched)
```

Comment:
Why do you modified this?

Reply:
After routing matching, the table will be cleared.

Reply:
I have decided to prioritize performance. 
udpate!